android/app/src/main/cpp/jitter_buffer.cpp

```
@@ -0,0 +1,113 @@
+        // the same payload; either way we don't gain by replacing).
+        return false;
+    }
+
+    Frame f;
+    f.seq = seq;
+    f.opusData.assign(data, data + size);
+    frames_.insert(it, std::move(f));
+    return true;
+}
+
+std::optional<JitterBuffer::Frame> JitterBuffer::pop() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)